download-large-logs.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo status:
+for LINKFILE in `find . -name "*.link"`; do
+    DARSHANFILE=${LINKFILE%.link}
+    echo "    ${DARSHANFILE}"
+    if [ -f $DARSHANFILE ] ; then
+        echo "        - already present, skipping."
+    else
+        echo "        - not present, downloading..."
+        wget -nv `cat $LINKFILE` -O $DARSHANFILE
```

Comment:
Of course, when I tried to run this on my mac laptop I got:

`./download-large-logs.sh: line 11: wget: command not found`

Since this is more of an advanced/bonus feature, I'm not sure a debate about the most portable solution/script language is really needed anyway. The download seems to work on my Linux box.

Reply:
Whoops.  I just pushed a commit that at least checks for wget up front and prints a friendlier message.  We could add support for other downloaders in a separate PR later if that seems helpful.